Doc/c-api/unicode.rst

```diff
@@ -387,7 +387,8 @@ APIs:
    arguments, calculate the size of the resulting Python Unicode string and return
    a string with the values formatted into it.  The variable arguments must be C
    types and must correspond exactly to the format characters in the *format*
-   ASCII-encoded string.
+   string. The *format* string is decoded from UTF-8 with the "strict" error
```

Comment:
You do not need to mention the "strict" handler explicitly, as it is default. *format* is now not different from most of other `const char *` parameters. You can just replace ASCII with UTF-8 here.

Reply:
Ok, I removed it.